_components/accordions.md

```diff
@@ -154,39 +184,35 @@ title: Accordions
         <button class="code-copy-button" data-clipboard-target="accordion-css">Copy</button>
         <div id="accordion-css">
           {% highlight css %}
-#accordion-menu ul {
+.usa-accordion ul {
   border: 1px solid #757575;
   border-radius: 5px;
   color: #212121;
   margin: 0;
   overflow: hidden;
-  padding: 0; 
+  padding: 0;
 }
 
-#accordion-menu ul li {
+.usa-accordion ul li {
   background-color: #eeeeee;
   border-bottom: 1px solid #757575;
   font-family: "Source Sans Pro", "Helvetica", "Arial", sans-serif;
   list-style: none;
-  margin: 0; 
-}
-
-#accordion-menu ul li:last-child {
-  border-bottom: 0; 
+  margin: 0;
 }
 
-#accordion-menu ul li.hidden .content {
-  display: none; 
+.usa-accordion ul li:last-child {
+  border-bottom: 0;
 }
 
-#accordion-menu ul li.hidden p {
-  background: url("https://cdn.rawgit.com/18F/govt-wide-patternlibrary/18f-pages-staging/assets/img/arrow-right.png") no-repeat;
+.usa-accordion ul button[aria-expanded=false] {
+  background: url("../img/arrow-right.png") no-repeat;
   background-position: 15px;
-  border-bottom: 0; 
+  border-bottom: 0;
 }
 
-#accordion-menu ul li p {
-  background: url("https://cdn.rawgit.com/18F/govt-wide-patternlibrary/18f-pages-staging/assets/img/arrow-down.png") no-repeat;
```

Comment:
Please put the correct reference back in so that users are able to use the image resource without having to download it.